images/ceph/Makefile

```diff
@@ -59,11 +59,6 @@ endif
 # s5cmd's version
 S5CMD_VERSION = 1.4.0
 
-VOL_REPL_VERSION = v0.1.0
```

Comment:
no need for pulling the crds in the makefile, this basically just moved to the build-crds.sh, which is closer to where they are needed.

Reply:
But now you're losing the ability to specify VOL_REPL_VERSION, and hardcoding the URL. This may be a problem if needing to use a different/newer location for whatever reason. Or am I missing something?

Reply:
The intention isn't to support different versions of the volume replication CRDs. When we intend to update the version, we would just change the hard coded version. So I don't think anything is lost in that sense.

Reply:
Sure, mostly I mean facilitating folks who may want to pull from different branches or forks.

Reply:
Really I'd just rather remove the csv permanently from the upstream build :)

Reply:
So say we all. :P

Reply:
Once this is addressed https://github.com/csi-addons/kubernetes-csi-addons/issues/116 we can remove it from the Rook

Reply:
Not a big deal for now, sounds good for the long term.